src/biogeochem/CNNDynamicsMod.F90

```diff
@@ -333,7 +333,7 @@ subroutine CNSoyfix (bounds, num_soilc, filter_soilc, num_soilp, filter_soilp, &
     associate(                                                                      & 
          wf               =>  waterdiagnosticbulk_inst%wf_col                      ,         & ! Input:  [real(r8) (:) ]  soil water as frac. of whc for top 0.5 m          
 
-         hui              =>  crop_inst%gddplant_patch                    ,         & ! Input:  [real(r8) (:) ]  gdd since planting (gddplant)                    
+         hui              =>  crop_inst%hui_patch                         ,         & ! Input:  [real(r8) (:) ]  patch heat unit index (growing degree-days)    
```

Comment:
Is HUI accumulated over a certain time period? GDD specifies that it's accumulated since planting, and I think it's the same for HUI. If so, does it make sense to add this description? Would something like "patch accumulated heat unit index (growing degree-days) since planting" be correct?

Reply:
You're right, a bit more info would be helpful. These descriptions now (10032f1) read (new text in bold), "crop patch heat unit index (growing degree-days)**; set to 0 at sowing and accumulated until harvest**".